arangod/V8Server/v8-collection.cpp

```diff
@@ -1076,9 +1076,9 @@ static void JS_FiguresVocbaseCol(
   OperationOptions options(ExecContext::current());
   auto opRes = collection->figures(details, options).get();
 
-  trx.finish(TRI_ERROR_NO_ERROR);
+  res = trx.finish(TRI_ERROR_NO_ERROR);
```

Comment:
In order to simplify things here, I think we don't need the call to `finish` here at all. It is a read-only transaction anyway, so nothing needs to be explicitly committed or aborted.

Reply:
I haven't removed it entirely, but still simplified it a little.